.readthedocs.yml

```diff
@@ -0,0 +1,13 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+python:
+  version: 3.7
```

Comment:
This is what readthedocs is currently using. We can upgrade to 3.8 at some point, but the goal here was just to record the existing settings in the repo